liquibase-couchbase/src/main/java/liquibase/ext/couchbase/operator/ClusterOperator.java

```diff
@@ -90,6 +90,14 @@ public List<String> retrieveDocumentIdsByWhereClause(Keyspace keyspace, String w
                 .collect(toList());
     }
 
+    public List<String> retrieveDocumentIdsBySqlPlusPlusQuery(String sqlPlusPlusQuery) {
```

Comment:
Parameter name could be just `sqlPlusPlusQuery`

Reply:
renamed to query